internal/runnerinstall/nomad.go

```diff
@@ -284,6 +288,12 @@ func (i *NomadRunnerInstaller) InstallFlags(set *flag.Set) {
 		Target: &i.Config.CsiSecrets,
 		Usage:  "Credentials for publishing volume for Waypoint runner.",
 	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "nomad-csi-volume",
+		Target: &i.Config.CsiVolume,
+		Usage:  fmt.Sprintf("The name of the volume to initialize within the CSI provider. The default is %s.", defaultRunnerName("<runner_id>")),
```

Comment:
When I tried to include docs with `<string>` it messed up the website generation because it thought `<string>` was an HTML tag. This is probably true here too, so maybe we could have it without the `<>`?

Reply:
Updated in https://github.com/hashicorp/waypoint/pull/3546/commits/819d3acdb202055ddd1c42e51e3b4403b63c6af1